### DISCUSSION TITLE
Calculation Groups in Bravo ?

### DISCUSSION THREAD

**Discussion by marcl2** [sql-bi/Bravo#767] [Ideas]
Dear Gents,
I cannot live without Bravo to reformat all my measures in one click. Excellent.
As the calculation groups are directly in PowerBI without TE, could it be possible to reformat also all calculation items in one click?
Thx in advance

**Comment by albertospelta** [comment]
We will consider adding DAX Formatter for calculation items in a future version of Bravo
Linking https://github.com/sql-bi/Bravo/issues/222 - please follow that issue for updates.

**Reply by marcl2** [reply]
Thx for the swift answer!
